Sources/SafeDICore/Models/TypeDescription.swift

```diff
@@ -44,8 +44,8 @@ public enum TypeDescription: Codable, Hashable, Comparable, Sendable {
     indirect case array(element: TypeDescription)
     /// A dictionary. e.g. [Int: String]
     indirect case dictionary(key: TypeDescription, value: TypeDescription)
-    /// A tuple. e.g. (Int, String)
-    indirect case tuple([TypeDescription])
+    /// A tuple. e.g. (Int, string: String)
+    indirect case tuple([TupleElement])
```

Comment:
Our generated `ForwardedArguments` tuples have labels, so we need our `TypeDescription` to know about labels now.